website/docs/r/api_gateway_documentation_part.html.markdown

```diff
@@ -0,0 +1,61 @@
+---
+layout: "aws"
+page_title: "AWS: aws_api_gateway_documentation_part"
+sidebar_current: "docs-aws-resource-api-gateway-documentation-part"
+description: |-
+  Provides a settings of an API Gateway Documentation Part.
+---
+
+# aws_api_gateway_documentation_part
+
+Provides a settings of an API Gateway Documentation Part.
+
+## Example Usage
+
+```hcl
+resource "aws_api_gateway_documentation_part" "example" {
+  location {
+    type = "METHOD"
+    method = "GET"
+    path = "/example"
+  }
+  properties = "{\"description\":\"Example description\"}"
+  rest_api_id = "${aws_api_gateway_rest_api.example.id}"
+}
+
+resource "aws_api_gateway_rest_api" "example" {
+  name = "example_api"
+}
+```
+
+## Argument Reference
+
+The following argument is supported:
+
+* `location` - (Required) The location of the targeted API entity of the to-be-created documentation part. See below.
+* `properties` - (Required) A content map of API-specific key-value pairs describing the targeted API entity. The map must be encoded as a JSON string, e.g., "{ \"description\": \"The API does ...\" }". Only Swagger-compliant key-value pairs can be exported and, hence, published.
+* `rest_api_id` - (Required) The ID of the associated Rest API
+
+### Nested fields
+
+#### `location`
+
+* `method` - (Optional) The HTTP verb of a method. It is a valid field for the API entity types of `METHOD`, `PATH_PARAMETER`, `QUERY_PARAMETER`, `REQUEST_HEADER`, `REQUEST_BODY`, `RESPONSE`, `RESPONSE_HEADER`, and `RESPONSE_BODY`. The default value is `*` for any method.
```

Comment:
Can we refer to the [API Gateway documentation](https://docs.aws.amazon.com/apigateway/api-reference/resource/documentation-part/) for this attribute and the ones below instead of listing these out?